worlds/witness/options.py

```diff
@@ -53,6 +63,61 @@ class ShuffleSymbols(DefaultOnToggle):
     display_name = "Shuffle Symbols"
 
 
+class ProgressiveSymbols(OptionSet):
+    """
+    Make some symbols progressive, if they exist.
+
+    By default, includes the chains where the second item can't be used without the first.
+
+    Progressive Dots: Dots -> Full Dots
+    Progressive Symmetry: Symmetry -> Colored Dots
+    Progressive Stars: Stars -> Stars + Same Colored Symbol
+    Progressive Squares: Black/White Squares -> Colored Squares
+    Progressive Shapers: Shapers -> Rotated Shapers -> Negative Shapers
+    Progressive Line-Counting Symbols: Triangles -> Arrows
+    """
+
+    valid_keys = {
+        "Progressive Dots",
+        "Progressive Symmetry",
+        "Progressive Stars",
+        "Progressive Squares",
+        "Progressive Shapers",
+        "Progressive Line-Counting Symbols"
+    }
+
+    default = {"Progressive Dots", "Progressive Symmetry", "Progressive Stars"}
+
+
+class SecondStageSymbolsActIndependently(Toggle):

```

Comment:
Maybe I missed it, but if none of these are progressive, this should throw an error/warning